loadtest/tests.go

```diff
@@ -108,6 +108,56 @@ func actionGetStatuses(c *EntityConfig) {
 	}
 }
 
+func actionLeaveJoinChannel(c *EntityConfig) {
+	team, channel := c.UserData.PickTeamChannel(c.r)
+
+	if team == nil || channel == nil {
+		return
+	}
+
+	channelId, err := c.GetTeamChannelId(team.Name, channel.Name)
+	if err != nil {
+		mlog.Error("Unable to get channel from map", mlog.String("team", team.Name), mlog.String("channel", channel.Name), mlog.Err(err))
+		return
+	}
+
+	userId := ""
+	if user, resp := c.Client.GetMe(""); resp.Error != nil {
+		mlog.Error("Failed to get me", mlog.Err(resp.Error))
+		return
+	} else {
+		userId = user.Id
+	}
+
+	if _, resp := c.Client.GetChannel(channelId, ""); resp.Error != nil {
+		mlog.Error("Failed to get channel", mlog.String("channel_id", channelId), mlog.Err(resp.Error))
+		return
+	}
+
+	removed, _ := c.Client.RemoveUserFromChannel(channelId, userId)
+
+	if removed {
+		time.Sleep(time.Second * 1)
```

Comment:
nit: `1 * time.Second` reads more clearly to me.

Reply:
Thanks for the feedback. That looks like the cleanest solution :+1: